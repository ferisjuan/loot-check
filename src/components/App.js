import React from 'react'
import Loot from './Loot'
import Wallet from './Wallet'

const App = () => {
	return (
		<div>
			<h1>Loot Check</h1>
			<hr />
			<Wallet />
			<hr />
			<Loot />
		</div>
	)
}

export default App
