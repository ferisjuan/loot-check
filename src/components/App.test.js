import React from 'react'
import { shallow } from 'enzyme'
import 'jest-enzyme'

import App from './App'

describe('App', () => {
	const app = shallow(<App />)

	it('Renders properly', () => {
		expect(app).toMatchSnapshot()
	})

	it('contains a wallet component', () => {
		// console.log(app.debug())
		expect(app.find('Connect(Wallet)').exists()).toBe(true)
	})
})
