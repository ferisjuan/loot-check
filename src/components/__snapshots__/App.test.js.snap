// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders properly 1`] = `
<div>
  <h1>
    Loot Check
  </h1>
  <hr />
  <Connect(Wallet) />
</div>
`;
